modules/swagger-codegen/src/main/resources/JavaPlayFramework/LICENSE.mustache

```diff
@@ -0,0 +1,8 @@
+This software is licensed under the Apache 2 license, quoted below.
```

Comment:
We might later need to empty this file as there's a discussion about not setting any license to the auto-generated code and let the users to decide which license (e.g. MIT, GNU, Apache2, etc) to use.